tests/Unit/Console/Command/AdminPromoteCommandTest.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace OpenCFP\Test\Unit\Console;
+namespace OpenCFP\Test\Unit\Console\Command;
```

Comment:
It covers `OpenCFP\Console\Command\AdminPromoteCommand`, so the corresponding unit test should reside in `OpenCFP\Test\Unit\Console\Command\AdminPromoteCommand`.